docker-compose.yml

```diff
@@ -1,21 +1,24 @@
 version: "3.9"
 services:
   paybutton:
+    depends_on: 
+      - mysql
```

Comment:
Adding in MySQL should have been its own branch, separated from the "add a paybutton" stuff.

https://ourmachinery.com/post/step-by-step-programming-incrementally/